projects/components/src/public-api.ts

```diff
@@ -266,6 +266,8 @@ export * from './popover/popover-ref';
 export * from './popover/popover.module';
 export * from './popover/popover.component';
 export * from './popover/popover.service';
+export * from './popover/popover-content.component';
```

Comment:
I don't think, we need to export it now. Do we?

Reply:
no we shouldn't need to, removing